pkg/collectors/nest/nest_test.go

```diff
@@ -29,6 +29,7 @@ func TestServerResponses(t *testing.T) {
 				HeatSetpointTemp: float64(19.17838),
 				Humidity:         float64(57),
 				Status:           "OFF",
+				Mode:             "HEAT",
```

Comment:
Please change to "HEATCOOL" to unbreak this test. "HEATCOOL" is what's in the mock Nest thermostat under test here: https://github.com/klyubin/pronestheus/blob/97bd3389ec4073db9c630a73f0abe1fcc36f38eb/test/testdata/nest_valid.json#L19